.circleci/config.yml

```diff
@@ -350,7 +350,9 @@ commands:
           name: Rename Reports
```

Comment:
Would it be possible to set the report name when we execute Playwright using `PLAYWRIGHT_BLOB_OUTPUT_NAME` or one of the other env vars listed [here](https://playwright.dev/docs/test-reporters#blob-reporter) instead, avoiding the need for this step.

Reply:
Yes, I tried that [here](https://github.com/mozilla/fxa/pull/17089), but it wasn't working for our `config.yaml` file somehow. I tried a number of things which accounts for the 55 commits in that PR :D 
And finally I had to resort to adding the 'renaming' step so that the `report.zip` file doesn't create a problem while merging, which it was (because of the same name).